### PR TITLE
Adding `mirrormaker2` home page and initial content

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Took the chance of migrating the #476 in order to create an home for MirrorMaker2. 
The #476 article contains **a lot** of information so I had to split it in several different articles, and I still need to add an article with a complete example with the Aiven cli

### CODE REVIEW

File: docs/products/kafka/kafka-mirrormaker/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Getting started
+===============
+
+Aiven services are managed in the Aiven `web console <https://console.aiven.io/>`__ . When you first log into the console with your email address and password, you will see the **Services** view, which shows you all the services in the currently selected project.
+
+.. Warning::
+
+    Aiven for Apache Kafka MirrorMaker2 services can be created only alongside at least one existing Aiven for Apache Kafka running services.
```
Comment (lornajane) [inline docs/products/kafka/kafka-mirrormaker/getting-started.rst:8]: At the end: .... running service

(to match the beginning of the sentence: at least one ..... running service)

**Comment by lornajane** [comment]
Structure looks good to me! I'd like either me or @TibsAtWork to do a language review before we merge this, but I'm also tagging as "help wanted" and asking for someone who knows MirrorMaker well to check the content before we polish and publish.

File: docs/products/kafka/kafka-mirrormaker/getting-started.rst (hunk 1 above)
Comment (ftisiot) [reply docs/products/kafka/kafka-mirrormaker/getting-started.rst:8]: Well spotted, thanks!

File: docs/products/kafka/kafka-mirrormaker/reference/terminology.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Terminology for Aiven for Apache Kafka MirroMaker2
```
Comment (frconil) [inline docs/products/kafka/kafka-mirrormaker/reference/terminology.rst:1]: Small typo:

```suggestion
Terminology for Aiven for Apache Kafka MirrorMaker2
```

File: docs/products/kafka/kafka-mirrormaker/reference/terminology.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+==================================================
+
+.. _Terminology MM2ClusterAlias:
+
+Cluster alias 
+    The name alias defined in MirroMaker2 for a certain Apache Kafka source or target cluster.
```
Comment (frconil) [inline docs/products/kafka/kafka-mirrormaker/reference/terminology.rst:7]:
```suggestion
    The name alias defined in MirrorMaker2 for a certain Apache Kafka source or target cluster.
```

**Comment by mdedetrich** [comment]
I did a quick skim of the content and everything to me seems good

**Comment by TibsAtWork** [comment]
Text looks excellent. Just going to add `®` to all the Kafka words before approving.

**Review by TibsAtWork — APPROVED** [review]
I believe the text is good.

**Pull request merged**